metecho/api/serializers.py

```diff
@@ -353,6 +353,7 @@ class Meta:
             "unsaved_changes",
             "total_unsaved_changes",
             "has_unsaved_changes",
+            "ignored_changes",
```

Comment:
@wlonk I'm assuming `ignored_changes` should be under the same privacy restrictions as `unsaved_changes`.